.rubocop.yml

```diff
@@ -2,7 +2,7 @@ require:
   - rubocop-performance
 
 inherit_from:
-  - https://raw.githubusercontent.com/RenoFi/rubocop/master/0.78.yml
+  - https://raw.githubusercontent.com/RenoFi/rubocop/master/0.80.yml
```

Comment:
`0.78.yml` didn't exist anymore, so I've changed it to `0.80.yml`, which is the only file available in the repo. With this step, I also needed to upgrade rubocop.